pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -144,3 +144,4 @@ func (op *overcommitPlugin) OnSessionClose(ssn *framework.Session) {
 	op.idleResource = nil
 	op.inqueueResource = nil
 }
+
```

Comment:
Can this blank line be deleted?
In addition, can some test cases be added to prove this pr?